### ISSUE TITLE
Add Evidence Room

### ISSUE THREAD

**Issue by HumanTree92** [ESX-Org/esx_policejob#83]
So i know on most servers you have a problem with corruption. Such as when an officer pulls a subject over & they have drugs on them. They Confiscate the items from him. But what does he do with these items. Maybe they are drugs. It is considered corrupt if you have an Officer selling the drugs or taking the drugs for himself. So in my Opinion there should be a Marker at the PD or an Option in the menu to allow the Officers to deposit these items into the PD as Evidence. 

I have found a way to do this but sadly it uses the put_stock. I just added the put_stock under the F6 Menu. Just would like to see what the devs of esx think of this & maybe make a full release of this.

**Comment by mrdigital0** [comment]
You can already do this buy using the police armory

**Comment by HumanTree92** [comment]
Yes while that is true but it still allows Regular Officers to remove stock. Meaning that if 1 cop puts drugs in the put_stock then another officer can go up there & take it out.

**Comment by SLuuuuuuG** [comment]
Just remove the option for them to withdrawal stock only deposit.

**Comment by HumanTree92** [comment]
Yes i see what you guys mean. However tho i don't use the armory management. This is supposed to mean a different menu like an Evidence Menu not Armory Menu.

**Comment by HumanTree92** [comment]
There are a few good locations if your at the Main PD with the Jail Cells. But a few locations i would have in mind would be either by the Desk where the Jail Cells are or behind the front desk. I would see it better being where the armory is. But if there was a way to combine the Armory with the Evidence menu it would be better.

**Comment by mrdigital0** [comment]
If i did code to combine the armory and the evidence i  would just keep it the same and do the get stock menu option for certain ranks that can remove

**Comment by HumanTree92** [comment]
I understand thats why i am saying its best to put in the basement of the PD. B/c the problem with the armory crap is anybody who is a cop can walk up & do remove_stock and they can take the drugs or whatever items they are and sell them or use them. Then you have a corrupt cop on your server & any server where cops are corrupt are not fun. Thats when you start to get people who troll the server.

**Comment by mrdigital0** [comment]
What I saying is lock down the remove_stock to a certain rank example
boss then only the Boss rank can remove

**Comment by ElPumpo** [comment]
Confiscating should remove the items, not give them to the player. Police shouldn't be corrupted, so that's the upcoming change

**Comment by ElPumpo** [comment]
We'll have to create a seperate inventory for the evidence storage...

**Comment by benzon** [comment]
+1 Agree that they should never get what's confiscated in there inventory - maybe add a feature to config, where you can fill out all illigal things and you can select confiscate illigal possesions, that way you dont have to manually click everything.

**Comment by SLuuuuuuG** [comment]
I setup my policejob so when they confiscate someone's items..the items just vanish into thin air.

**Comment by Ultrunz** [comment]
There is a map with an evidence room. You'd just need to put a storage box in there with locks on the doors. I think I have it on my esx_doorlock fork I'll have to check.

**Comment by HumanTree92** [comment]
+1 with @benzon 
I agree that having a config list of illegal items should be added. But also the confiscate dirty money should be fixed as well. B/c if you confiscate dirty money it gives you the dirty money.

**Comment by ElPumpo** [comment]
Yea cops should not be able too see your entire inventory
- there should be a button saying 'confiscate illegal items
- confiscating should not go to cops invrntory, instead go to evidence room

**Comment by ElPumpo** [comment]
An update on this: Ive started working on the new code for this. It will be kinda what benzon suggested: Instead of selecting what to confiscate you'll just have a configurable list of illegal items & weapons. Upon seizing you'll take whats illegal, and searching player will reveil all illegal stuff the player is carrying. Seizing items will delete them
